courses/free-courses-en.md

```diff
@@ -553,6 +553,7 @@
 * [Introduction to Reinforcement learning with David Silver](https://www.youtube.com/playlist?list=PLqYmG7hTraZBiG_XpjnPrSNw-1XQaM_gB) - David Silver
 * [MIT 6.S094: Deep Learning for Self-Driving Cars](https://selfdrivingcars.mit.edu)
 * [MIT 6.S191: Introduction to Deep Learning](http://introtodeeplearning.com) - Alexander Amini, Ava Soleimany
+* [NLP Course](https://huggingface.co/learn/nlp-course/chapter1/1) - Hugging Face
```

Comment:
```suggestion
* [Hugging Face NLP Course](https://huggingface.co/learn/nlp-course/chapter1/1) - Hugging Face
```
somtimes you need to see what the site itself calls the course. In this case, I looked at the `<title>`

Reply:
I've changed the title to Hugging Face NLP Course.